### PR TITLE
test: plugin-rsc-analyze

### PR DESCRIPTION
**Pull request by himself65**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@ jobs:
       - uses: pnpm/action-setup@v2
       - uses: actions/setup-node@v3
         with:
-          node-version: 18
+          node-version: 20
```
Comment (himself65) [inline .github/workflows/ci.yml:21]: nodejs 18 doesn't have global.crypto?

File: packages/website/private/README.md [hunk 2]
```diff
@@ -1 +1 @@
-../../../README.md
\ No newline at end of file
+../../../README.md
```
Comment (dai-shi) [inline packages/website/private/README.md:1]: Can you revert the change?

**Review by dai-shi — APPROVED** [review]
The structure looks good to me.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (dai-shi) [reply .github/workflows/ci.yml:21]: #760
